shoop/core/models/_contacts.py

```diff
@@ -281,6 +293,12 @@ def is_all_seeing(self):
         if self.user_id:
             return getattr(self.user, 'is_superuser', False)
 
+    def get_orders(self):
+        order_q = Q(orderer=self) | Q(customer=self)
```

Comment:
Are you sure you want to mix these together. IMO you don't need this get_orders. You can already get already get the orders for contact throug related manager. And you should probably always use customer orders, orderer_orders and orders_created separately.